code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -7,15 +7,30 @@
 	slot_flags = SLOT_BELT
 	force = 10
 	throwforce = 7
-	w_class = WEIGHT_CLASS_NORMAL
 	origin_tech = "combat=2"
 	attack_verb = list("beaten")
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 50, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
+	/// How many life ticks does the stun last for
 	var/stunforce = 7
-	var/status = 0
-	var/obj/item/stock_parts/cell/high/cell = null
+	/// Is the baton currently turned on
+	var/enabled = FALSE
+	/// How much power does it cost to stun someone
```

Comment:
Also generally what even is this documentation for? It's not like this is frequently reused logic. Nothing wrong with adding some extra documentation, but time is money.

Reply:
If it's something obvious like `cell` then I'll just leave it, but all of the others except `enabled` benefit from some sort of explanation.
And even with `enabled`, it can't hurt to make it idiot proof.